### DISCUSSION TITLE
Error: DatabaseEntry with ID '12' already exists

### DISCUSSION THREAD

**Discussion by jruben4** [Koenkk/zigbee2mqtt#12316] [General]
My zigabee2mqtt container seems to stop working after a few hours.  When I check, the container is stopped, and this is the error in the log:

Error: DatabaseEntry with ID '12' already exists

I'm running it on Unraid with a zTube ethernet coordinator.

**Comment by pjcarly** [comment]
I'm having similar issues, any idea what causes it?

**Comment by RenHoekNL** [comment]
Same here

> Zigbee2MQTT:info  2023-02-11 15:47:19: Successfully interviewed '0xa4c138e39fec7045', device has successfully been paired
> Zigbee2MQTT:info  2023-02-11 15:47:19: Device '0xa4c138e39fec7045' is supported, identified as: TuYa Smart plug (with power monitoring) (TS011F_plug_1)
> Zigbee2MQTT:info  2023-02-11 15:47:19: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"Smart plug (with power monitoring)","expos..<snip>.....":"numeric"}],"supports_ota":true,"vendor":"TuYa"},"friendly_name":"0xa4c138e39fec7045","ieee_address":"0xa4c138e39fec7045","status":"successful","supported":true},"type":"device_interview"}'
> Zigbee2MQTT:info  2023-02-11 15:47:19: Configuring '0xa4c138e39fec7045'
> Error: DatabaseEntry with ID '3' already exists
>     at Database.insert (D:\zigbee2mqtt\node_modules\zigbee-herdsman\src\controller\database.ts:40:19)
>     at Device.undelete (D:\zigbee2mqtt\node_modules\zigbee-herdsman\src\controller\model\device.ts:394:25)
>     at Controller.onDeviceJoined (D:\zigbee2mqtt\node_modules\zigbee-herdsman\src\controller\controller.ts:543:20)

This was due to unplugging and plugging back in a zigbee power socket

**Comment by jc-plhm** [comment]
same here, using with ezsp adapter

`
zigbee-herdsman:controller:log Device '0x84fd27fffe932847' accepted by handler +1ms

zigbee-herdsman:controller:log Delete device '0x84fd27fffe932847' joined, undeleting +0ms

Error: DatabaseEntry with ID '4' already exists
    at Database.insert (/app/node_modules/zigbee-herdsman/src/controller/database.ts:40:19)
    at Device.undelete (/app/node_modules/zigbee-herdsman/src/controller/model/device.ts:396:25)
    at Controller.onDeviceJoined (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:543:20)
`

happens sometimes when removing a device outside the network and rejoining it.

**Comment by mattheworiordan** [comment]
Same issue for me too.

```
Error: DatabaseEntry with ID '9' already exists
    at Database.insert (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/database.ts:40:19)
    at Device.undelete (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/model/device.ts:394:25)
    at Controller.onDeviceJoined (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/controller.ts:543:20)
```

**Comment by roynasser** [comment]
Having the same issue migrating from ZHA to Z2M... add on is crashing in home assistant
